### DISCUSSION TITLE
基于 Docker Compose 进行独立部署，启动失败

### DISCUSSION THREAD

**Discussion by w123wh** [higress-group/higress#1196] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/e0c027ff-fa0a-4a08-a973-e3cbc8add809)

**Comment by w123wh** [comment]
higress-apiserver-1日志
![image](https://github.com/user-attachments/assets/1e1971ec-c8ba-4846-a38d-2896e9a7db18)

**Comment by CH3CHO** [comment]
是不是在一个 nacos 上覆盖安装了？先清理一下 nacos 里的配置重新安装一遍。

**Reply by w123wh** [reply]
我已尝试删除相关配置

**Reply by CH3CHO** [reply]
还不行吗？

**Reply by w123wh** [reply]
还是不行

**Reply by w123wh** [reply]
关于清理一下 nacos 里的配置，是否有具体操作，我是删除了higress-system命名空间下所有配置

**Reply by CH3CHO** [reply]
对，删除就行，但不能直接删命名空间，要把配置删了。

你清空配置之后，还是一样的错误吗？

**Reply by CH3CHO** [reply]
如果不行的话，你用内置的nacos启动试试看

**Reply by w123wh** [reply, marked as answer]
我删除了持久化配置现在可以了，但是我遇到的新的问题，higress配置域名但是无法访问502

**Reply by CH3CHO** [reply]
先确认502是谁返回的。你可以针对这个问题单独提一个discussion。

**Reply by w123wh** [reply]
我已解决，十分感谢
